apps/e2ee-server/src/utils/cryptoUtils.ts

```
@@ -0,0 +1,236 @@
+      'hex',
+    );
+  }
+
+  /**
+   * Verify data integrity
+   * @param data Original data
+   * @param hash Expected hash value
+   * @returns Whether data is intact
+   */
+  static verifyHash(data: string, hash: string): boolean {
+    const computedHash = this.createHash(data);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)